Mathlib/Analysis/Meromorphic/TrailingCoefficient.lean

```diff
@@ -207,8 +207,8 @@ lemma MeromorphicAt.meromorphicTrailingCoeffAt_smul {f₁ : 𝕜 → 𝕜} {f₂
   have : f₁ • f₂ =ᶠ[𝓝[≠] x]
       fun z ↦ (z - x) ^ (meromorphicOrderAt (f₁ • f₂) x).untop₀ • (g₁ • g₂) z := by
     filter_upwards [h₃g₁, h₃g₂, self_mem_nhdsWithin] with y h₁y h₂y h₃y
-    simp_all [meromorphicOrderAt_smul hf₁ hf₂]
-    rw [← smul_assoc, ← smul_assoc, smul_eq_mul, smul_eq_mul, zpow_add₀ (sub_ne_zero.2 h₃y)]
+    simp_all [meromorphicOrderAt_smul hf₁ hf₂, ← smul_assoc, ← smul_assoc, smul_eq_mul,
+      smul_eq_mul, zpow_add₀ (sub_ne_zero.2 h₃y)]
     ring_nf
```

Comment:
Does `module` help at all here?

Reply:
Yes: it works instead of `ring_nf` and is faster (and also allows removing some manually provided lemmas). Thanks for the suggestion!